upi/openstack/compute-nodes.yaml

```diff
@@ -86,5 +81,6 @@
       userdata: "{{ lookup('file', 'worker.ign') | string }}"
       nics:
       - port-name: "{{ os_port_worker }}-{{ item.0 }}"
+      meta: "{{ cluster_id_tag }}"
```

Comment:
Are we should that this should go here and not above L71?

Reply:
Yes , It should go to L71. Will add that.